style/modules/gridBox.css

```diff
@@ -5,7 +5,8 @@
 
 .gridBox > * {  /* applies to all the items in the grid */
     vertical-align: top;
-    margin: 5px; /* the space between blocks. Doesn't affect on the components inside the blocks */
+    margin: 12px;
```

Comment:
I removed a comment here that said "doesn't affect the components inside" because if they have margins, it will affect that.